src/state/store.ts

```diff
@@ -47,6 +45,23 @@ export const clearState = (opts?: { excludePreferences?: boolean }) => {
   store.dispatch(apiSlices.stakingDataApi.util.resetApiState())
 }
 
+const actionSanitizer = (action: any) => {
+  const blackList = [
+    'asset/setAssets',
+    'assetApi/executeQuery/fulfilled',
+    'marketData/setMarketData',
+    'marketData/setPriceHistory',
```

Comment:
these are large actions and make the redux devtools crash, blacklist them from the devtools (doesn't actually change state)